packages/quantic/cypress/e2e/default-1/sort/sort-expectations.ts

```diff
@@ -47,6 +47,10 @@ function sortExpectations(selector: SortSelector) {
         expect(customData).to.have.property('resultsSortBy', value);
       });
     },
+
+    displayErrorMessage: (display: boolean) => {
+      selector.errorMessage().should(display ? 'exist' : 'not.exist');
+    },
```

Comment:
When you use the QuanticComponentError, you will be able to us the `ComponentErrorExpectations` declared in the `common-expectations.ts` file.